internal/seclang/directives.go

```diff
@@ -646,9 +646,9 @@ func directiveSecAuditLogType(options *DirectiveOptions) error {
 	return nil
 }
 
-// Description: Select the output format of the AuditLogs. The format can be either
-// the native AuditLogs format or JSON.
-// Syntax: SecAuditLogFormat JSON|Native
+// Description: Select the output format of the AuditLogs. The format can be
+// the native AuditLogs format, JSON, or OCSF (Open CyberSecurity Schema Framework).
+// Syntax: SecAuditLogFormat JSON|Native|OCSF
```

Comment:
For completeness' sake, let's also add here the `JsonLegacy`. These are the comment lines from where the website pages are generated. 
```suggestion
// Syntax: SecAuditLogFormat JSON|JsonLegacy|Native|OCSF
```